### DISCUSSION TITLE
JitPack broken ?

### DISCUSSION THREAD

**Discussion by jeffmaury** [jbangdev/jbang#766] [Q&A]
I had a script using a JitPack dependency:

//DEPS https://github.com/jgwest/zenhub-api-java-client

Used to work but jbang is now failing to resolve the dependency

**Comment by quintesse** [comment]
What I see is that it's trying to retrieve `HEAD-SNAPSHOT` instead of `master-SNAPSHOT`. IIRC there was a commit some time ago that changed the use of `master` to `HEAD` for some situations some time ago. That probably broke this?

**Reply by quintesse** [reply]
But I can confirm this is a bug so perhaps you can open an issue for this @jeffmaury ?

**Reply by quintesse** [reply]
https://github.com/jbangdev/jbang/commit/0a93f0c111afa6af839f171803e7cec3a32b4e2c is the commit I was referring to.

**Reply by quintesse** [reply]
Btw @maxandersen , I'm not entirely sure but for JitPack it might be possible to use `-SNAPSHOT` to get the version from the main branch, whatever it's named: https://jitpack.io/#jgwest/zenhub-api-java-client/-SNAPSHOT

**Reply by maxandersen** [reply]
I tested Head back then. So im surprised that would break it.

**Reply by quintesse** [reply]
But did you test it with JitPack? Because it does indeed work with GitHub, GitLab and BitBucket

**Reply by maxandersen** [reply]
yes, our tests does too.. at least as far as i remember :)

**Reply by maxandersen** [reply]
let me check.

**Reply by maxandersen** [reply]
hmm - it does break i can see that. im pretty sure i tested it but it definitely donmt work now :(

**Reply by maxandersen** [reply]
suggestion for fix at https://github.com/jbangdev/jbang/pull/767

please test/review
